tests/unit/sagemaker/lineage/test_query.py

```diff
@@ -524,3 +525,53 @@ def test_vertex_to_object_unconvertable(sagemaker_session):
 
     with pytest.raises(ValueError):
         vertex.to_lineage_object()
+
+
+def test_get_visualization_elements(sagemaker_session):
+    lineage_query = LineageQuery(sagemaker_session)
+    sagemaker_session.sagemaker_client.query_lineage.return_value = {
+        "Vertices": [
+            {"Arn": "arn1", "Type": "Endpoint", "LineageType": "Artifact"},
+            {"Arn": "arn2", "Type": "Model", "LineageType": "Context"},
+        ],
+        "Edges": [{"SourceArn": "arn1", "DestinationArn": "arn2", "AssociationType": "Produced"}],
+    }
+
+    query_response = lineage_query.query(
+        start_arns=["arn:aws:sagemaker:us-west-2:0123456789012:context/mycontext"]
+    )
+
+    elements = query_response._get_visualization_elements()
+
+    assert elements["nodes"][0] == ("arn1", "Endpoint", "Artifact", False)
+    assert elements["nodes"][1] == ("arn2", "Model", "Context", False)
+    assert elements["edges"][0] == ("arn1", "arn2", "Produced")
```

Comment:
Should there be a `True` value for the 4th element here?

Reply:
`True` and `False` only appears on `node` elements
I took reference from the previous query tests and it sets the `return_value` of `query_lineage` and queries it by using `arn:aws:sagemaker:us-west-2:0123456789012:context/mycontext` as start arn. What's weird is the startarn won't appear, so thats why there isn't a `True` on nodes

Reply:
Should I use the `LineageResourceHelper` I created from the helper in `integ` tests? But this test will be ran in the `unit` test

Reply:
Got it.

I don't think you need it here, since the unit tests are designed to not create resources (as that would be a more end-to-end/integ test)